### PR TITLE
Revised config example

### PR DESCRIPTION
**Pull request by sabrinajess**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1937663
For 4.6 and later

Preview: https://deploy-preview-31680--osdocs.netlify.app/openshift-enterprise/latest/virt/live_migration/virt-live-migration-limits.html

### CODE REVIEW

**Comment by sabrinajess** [comment]
@jeniferh Updated the fix, thanks for catching that. Please review.

**Comment by jeniferh** [comment]
LGTM!

File: modules/virt-configuring-live-migration-limits.adoc [hunk 1]
```diff
@@ -25,18 +25,32 @@ $ oc edit configmap kubevirt-config -n openshift-cnv
 [source,yaml]
```
Comment (adellape) [inline modules/virt-configuring-live-migration-limits.adoc:25]: Possible spot for an `.Example output` [block title](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output)?

File: modules/virt-configuring-live-migration-limits.adoc [hunk 2]
```diff
@@ -25,18 +25,32 @@ $ oc edit configmap kubevirt-config -n openshift-cnv
...
+metadata:
+  creationTimestamp: "2021-03-26T18:01:04Z"
+  labels:
+    app: kubevirt-hyperconverged
+  name: kubevirt-config
+  namespace: openshift-cnv
+  resourceVersion: "15371295"
+  selfLink: /api/v1/namespaces/openshift-cnv/configmaps/kubevirt-config
+  uid: <uuid>
+
```
Comment (adellape) [inline modules/virt-configuring-live-migration-limits.adoc:55]: Looks like it doesn't show up in the rendered version, but did notice an extra blank line here in source.

**Review by adellape — COMMENTED** [review]
Looks cool!

File: modules/virt-configuring-live-migration-limits.adoc (hunk 1 above)
Comment (sabrinajess) [reply modules/virt-configuring-live-migration-limits.adoc:25]: Added block title  `.Example configuration file`

File: modules/virt-configuring-live-migration-limits.adoc (hunk 2 above)
Comment (sabrinajess) [reply modules/virt-configuring-live-migration-limits.adoc]: Removed.

**Pull request merged**